### PR TITLE
CLN: misc cleanups from LGTM.com

### PR DESCRIPTION
**Pull request by simonjayhawkins**

### CODE REVIEW

**Comment by jbrockmendel** [comment]
Nice, LGTM pending green

File: pandas/core/frame.py [hunk 1]
```diff
@@ -8218,7 +8218,6 @@ def blk_func(values):
                     result = result.iloc[0].rename(None)
                 return result
 
-        data = self
```
Comment (topper-123) [inline pandas/core/frame.py:8221]: The ``data = self`` at line 8247 could be collapsed together with the line below, so we get ``values = self.values``.

**Review by topper-123 — APPROVED** [review]
A single comment else LGTM

File: pandas/core/frame.py (hunk 1 above)
Comment (simonjayhawkins) [reply pandas/core/frame.py:8221]: data is used further down on L8263

Comment (topper-123) [reply pandas/core/frame.py:8221]: Right, missed that. LGTM.

**Comment by jbrockmendel** [comment]
Thanks @simonjayhawkins

**Pull request merged**